test/dummy-instantiation.ts

```diff
@@ -0,0 +1,12 @@
+import { SafeCreationSettings, BridgeParameter } from "../src/ts";
+
+export const dummyteamControllerSettings: SafeCreationSettings = {
```

Comment:
I don't see it being used in this PR? Do I miss anything?

Reply:
Good catch! It was previously called `dummyteamConrollerSettings` (notice the typo), which is why you don't see it was removed. I fixed the typo early and I never realized that it's not needed anymore.